### ISSUE TITLE
Add disabled option

### ISSUE THREAD

**Issue by ilanolkies** [nzambello/react-csv-reader#30]
Hi!

I think a good feature would be adding `disabled` prop for `CSVReader`.

It can be used like
```javascript
<CSVReader onFileLoaded={this.handleUploadFile} disabled={validating} />
```

Cheers!

**Comment by nzambello** [comment]
@ilanolkies Released in v2.2.0 👍

**Comment by ilanolkies** [comment]
Awesome! Thanks. Will be very useful.
